[Mod] Hidden error in slowmode with large ints

# Command bugs

#### Command name

`[p]slowmode 10000000000000000000000`
Technically this is an issue with `commands.TimedeltaConverter`'s handling of large ints

#### What cog is this command from?

Mod

#### What happened?

Nothing in chat, this error in console
```py
[2020-06-26 02:30:54] [ERROR] red: ConversionError
Traceback (most recent call last):
  File "\redbot\core\commands\converter.py", line 287, in convert
    delta = parse_timedelta(
  File "\redbot\core\commands\converter.py", line 115, in parse_timedelta
    delta = timedelta(**params)
OverflowError: Python int too large to convert to C int
```

--- a/redbot/core/commands/converter.py
+++ b/redbot/core/commands/converter.py
@@ -112,7 +112,12 @@ def parse_timedelta(
                     _("`{unit}` is not a valid unit of time for this command").format(unit=k)
                 )
         if params:
-            delta = timedelta(**params)
+            try:
+                delta = timedelta(**params)
+            except OverflowError:
+                raise BadArgument(
+                    _("The time set is way too high, consider setting something reasonable.")
+                )
             if maximum and maximum < delta:
                 raise BadArgument(
                     _(

